internal/compiler-interface/src/main/contraband/other.json

```diff
@@ -28,6 +28,24 @@
             "Combined with a way to tell if the provenance has changed,",
             "it can be used to short-circuit the 'lookupAnalyzedClass' operation."
           ]
+        },
+        {
+          "name": "bytecodeHash",
+          "type": "int",
+          "default": "0",
+          "since": "2.0.0",
+          "doc": [
+            "A hash of generated bytecode of source file hosting the class"
+          ]
+        },
+        {
+          "name": "extraBytecodeHash",
```

Comment:
"extra" often means something supplied by the build user. I suggest `transitiveBytecodeHash` instead.